ios/test_app.rb

```diff
@@ -80,13 +92,18 @@ def project_path(file, target_platform)
   File.expand_path(file, File.join(__dir__, '..', target_platform.to_s))
 end
 
-def react_native_path(target_platform)
-  react_native = case target_platform
-                 when :ios then 'react-native'
-                 when :macos then 'react-native-macos'
-                 else raise "Unsupported target platform: #{target_platform}"
-                 end
-  Pathname.new(resolve_module(react_native))
+def react_native_path(project_root, target_platform)
+  react_native_from_manifest = react_native_path_from_manifest(project_root, target_platform)
+  if !react_native_from_manifest.nil?
+    react_native_from_manifest
```

Comment:
Prefer to return early:
```suggestion
  return react_native_from_manifest if react_native_from_manifest.is_a? String
```